src/libAtomVM/memory.c

```diff
@@ -333,8 +338,13 @@ static void memory_scan_and_copy(term *mem_start, const term *mem_end, term **ne
                     TRACE("- Found float.\n");
                     break;
 
-                case TERM_BOXED_REFC_BINARY:
-                    TRACE("- Found binary.\n");
+                case TERM_BOXED_REFC_BINARY: {
+                    TRACE("- Found refc binary.\n");
+                    term ref = ((term) ptr) | TERM_BOXED_VALUE_TAG;
+                    if (!term_refc_binary_is_const(ref)) {
+                        *mso_list = term_list_init_prepend(ptr + term_refc_binary_mso_offset(), ref, *mso_list);
```

Comment:
well, maybe here a `#define` was enough. Sorry if I wasn't clear enough.

Reply:
Changed to a #define'd constant.